app/collections/home.cjsx

```diff
@@ -0,0 +1,45 @@
+React = require 'react'
+CollectionCreateForm = require './create-form'
+talkClient = require '../api/talk'
+apiClient = require '../api/client'
+authClient = require '../api/auth'
+auth = require '../api/auth'
+{Link} = require 'react-router'
+ChangeListener = require '../components/change-listener'
+PromiseRenderer = require '../components/promise-renderer'
+
+module?.exports = React.createClass
+  displayName: 'CollectionsHome'
+
+  getInitialState: ->
+    collections: []
+
+  componentWillMount: ->
+    @setCollections()
+
+  setCollections: ->
+    auth.checkCurrent().then (user) =>
+      user.get('collections')
+        .then (collections) =>
+          @setState {collections}
+        .catch (e) -> throw new Error(e)
+
+  collectionLink: (d, i) ->
+    <div key={i}>
```

Comment:
I'll change these to key on resource id. 

As an aside: Why call keying on indexes bad news? React.js does it in their examples, internally when no key is supplied, and recommends it over random numbers